tests/test_markup.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+def test_good_markup():
+    import manimpango  # noqa: F401
+
+    assert manimpango.MarkupUtils.validate(
+        "foo"
+    ), '"foo" should not fail validation'
```

Comment:
I know this is frustrating, this is because of flake8+black combination. Using `\` will simply work.
I will commit it :)

Reply:
I see... they work together nicely :) Thanks